app/theme/spriteImage.component.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { mobile, desktop } from './sprites';
+
+const mobileSprite = (props) => {
+  const name = props.mobile || props.name;
+  if (name) {
+    return mobile(name);
+  }
+
+  return null;
+};
+
+const desktopSprite = (props) => {
```

Comment:
What do  you think about add also tablet sprite here?

Reply:
Well, I've removed it, since not all projects need tablet sprites :)